presto-orc/src/test/java/com/facebook/presto/orc/metadata/TestOrcType.java

```diff
@@ -73,4 +76,41 @@ public void testMapColumnToNodeIgnoreMissingColumns()
         Set<Integer> expected = ImmutableSet.of(1);
         assertEquals(actual, expected);
     }
+
+    @Test
+    public void testCreateNodeIdToColumnIdMapWithMultipleColumns()
+    {
+        List<String> columnNames = ImmutableList.of("f1", "f2", "f3", "f4");
+        List<Type> columnTypes = ImmutableList.of(VARCHAR, OrcTester.mapType(VARCHAR, VARCHAR), VARCHAR, OrcTester.mapType(VARCHAR, OrcTester.arrayType(VARCHAR)));
+        List<OrcType> orcTypes = OrcType.createOrcRowType(0, columnNames, columnTypes);
+        Map<Integer, Integer> actual = createNodeIdToColumnIdMap(orcTypes);
+        ImmutableMap<Object, Object> expected = ImmutableMap.builder()
```

Comment:
ImmutableMap > Map

Reply:
I didn't follow this. Can you elaborate?

Reply:
Use type Map instead of ImmutableMap for the `expected` variable.

Reply:
Ahh okay.